src/server/main_service.cc

```diff
@@ -1511,13 +1521,6 @@ void Service::Exec(CmdArgList args, ConnectionContext* cntx) {
   }
 
   ExecEvalState state = DetermineEvalPresense(exec_info.body);
-  if (state == ExecEvalState::SOME) {
```

Comment:
where do you test it now?

Reply:
Nowhere, but I don't run squashing with it for sure.